.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
not sure what those `.idea` files are but they probably shouldn't be commited to the repo.

Reply:
Will do that.